src/UnoFlipController.java

```diff
@@ -86,42 +105,62 @@ public void actionPerformed(ActionEvent e) {
                         null,
                         null);
 
-                Card.Colour colour;
-                if (result == JOptionPane.OK_OPTION) {
+                // Colour is selected and OK is pressed
+                if (result == JOptionPane.OK_OPTION)
+                {
+                    // Colour to set the new current colour of the game
+                    Card.Colour colour;
+                    // Parse the input given by the user from the drop-down to set the new colour
                     String option = (String) comboBox.getSelectedItem();
                     colour = Card.Colour.valueOf(option);
+                    // Sets the new current colour
                     this.model.setCurrentColour(colour);
+                    // Notify view to show status of the current colour
                     this.model.notifyViews();
                     break;
-                } else if (result == JOptionPane.CLOSED_OPTION){
-                    continue;
                 }
             }
 
-        } else if(e.getActionCommand().equals(UnoFlipViewFrame.DRAW_CMD)){
+        }
+        // Draw card command for the player to draw a card
```

Comment:
This can be better described as "The "draw card" button was pressed". Also, move this comment inside the the conditional block, I think it makes sense to move comments that explain "what happened" (past tense) inside the block of a conditional that indicates that the action has happened, when the conditional is easy to read.